aws/mysql/mysql.tf

```diff
@@ -79,6 +79,7 @@ module "rds" {
   major_engine_version = "8.0"
   instance_class       = "db.t2.micro"
   allocated_storage    = 10
+  storage_encrypted    = false
```

Comment:
Now it's `true` by default. I think we probably also should have it `true` by default but decided to set it to `false` for now because it might break something for the existing infrastructures. Let me know if we can set it to `true` by default.